### PR TITLE
Update bug_report.md

### PR DESCRIPTION
**Pull request by raakesh-blokhra**
Need information about whether the instance is connecting to Apigee X or Apigee Edge

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -32,3 +32,5 @@ This can be the version you can see on admin/modules in Drupal or
 the output of this command: `composer show`.  Add Drupal core and
 other version information if needed.
 
+## Organization Info
```
Comment (giteshk) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Can we call this Apigee Info

and move this as the second question after description?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -32,3 +32,5 @@ This can be the version you can see on admin/modules in Drupal or
...
+Please specify if you are using Apigee X or Apigee Edge.
```
Comment (giteshk) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Can we add OPDK to this list?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (raakesh-blokhra) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

**Review by giteshk — APPROVED** [review]
LGTM

**Pull request merged**